perfmetrics/scripts/load_tests/python/load_generator/constants.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for constants to be used in load generator module and script.
+"""
+# Sizes
+KB = 1024
+MB = 1024 * KB
+GB = 1024 * MB
+
+TIME_LOADING_PERCENTAGES = (0.25, 0.50, 0.75)
```

Comment:
Can you please add a one liner comment of it's use case?

Reply:
Added.